controllers/front/ProductController.php

```diff
@@ -441,30 +441,6 @@ public function displayAjaxRefresh()
         return;
     }
 
-    /**
-     * Get minimal product quantity or minimal product combination quantity.
-     *
-     * @param $product
-     *
-     * @return int
-     */
-    protected function getMinimalProductOrDeclinationQuantity($product)
```

Comment:
Should be marked as deprecated instead

Reply:
since it's the only place where it is used, I don't see the interest of keeping it
it only increases the risk that somebody else tries to use it when it doesn't even work

Reply:
it's an old controller and it allows override :/ Someone is maybe using it/

Reply:
sh*** ^^ I forgot about the override Ok I put it back with a deprecated annotation